### PR TITLE
salati app

### PR DESCRIPTION
**Pull request by seif1000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ A curated list of awesome things and projects built by Algerian developers.
 - [Salat Time](https://github.com/Barbarosse/salat_time/) A simple python Web App that returns the five Islamic Prayer Times.
 - [selma](https://github.com/FdevTech/Selma) - A virtual assistant backed by augmented reality capabilities, it allows you to control different appliances in your home.
 - [Timgad](https://github.com/OpenDZ/timgad) - Timgad is a kernel Linux Security Module that collects per process and system-wide security protections that are not handled by the core kernel itself.
-
+- [salati](https://github.com/seif1000/salati) an android app that helps muslims to performe their prayers in time by reminding them every day when the time comes  by adhan alarm .
```
Comment (youknowriad) [inline README.md]: Hi There! Thanks for the addition, I saw that your project is missing a LICENSE file, can you fix that?

Also, would you be able to put this on the right alphabetical order spot (after Salat Time)

Comment (CaddyDz) [inline README.md]: helps Muslims perform*

Comment (CaddyDz) [reply README.md]: also, the extra space between "comes" and "by" is an abomination in my eyes xD ¯\\_(ツ)_/¯

Comment (seif1000) [reply README.md]: I fixed it  .

File: README.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ A curated list of awesome things and projects built by Algerian developers.
 - [LSQLApp](https://github.com/houarizegai/lsqlapp) - A desktop application that helps teachers in practical work (TP) to evaluate students. It also helps students to learn SQL.
 - [OpenClass](https://github.com/youben11/open-class/) - OpenClass is a web app that promotes information sharing through organized workshops.
 - [Salat Time](https://github.com/Barbarosse/salat_time/) A simple python Web App that returns the five Islamic Prayer Times.
+- [salati](https://github.com/seif1000/salati) an android app that helps muslims to perform their prayers in time by reminding them every day when the time comes by adhan alarm .
```
Comment (youknowriad) [inline README.md]: Small wording suggestion

```suggestion
- [salati](https://github.com/seif1000/salati) an android app that helps muslims perform their prayers in time by reminding them every day when the time comes with an adhan alarm.
```

**Review by youknowriad — APPROVED** [review]
Looks good

**Pull request merged**